docs/admin/release_notes/version_1.12.md

```diff
@@ -0,0 +1,19 @@
+# v1.12 Release Notes
```

Comment:
Can you update mkdocs.yml file to reflect this page?

Reply:
Yep. Sorry I forgot about that.